app/constants/constants.js

```diff
@@ -194,6 +197,28 @@ export const CONNECTION_CONFIG = {
             'type': 'password',
             'description': 'Your data.world read/write token. It can be obtained from https://data.world/settings/advanced'
         }
+    ],
+    [DIALECTS.ATHENA]: [
+        {
+            'label': 'S3 Access Key', 'value': 'accessKey', 'type': 'password'
+        },
+        {
+            'label': 'S3 Secret Access Key', 'value': 'secretKey', 'type': 'password'
+        },
+        {
+            'label': 'AWS Region', 'value': 'region', 'type': 'text'
+        },
+        {
+            'label': 'S3 Bucket', 'value': 's3Outputlocation', 'type': 'text',
+            'description': 'The Athena will put the output results of the query in this location.'
+        },
+        {
+            'label': 'Database', 'value': 'database', 'type': 'text'
+        },
+        {
+            'label': 'Athena QueryTimeout', 'value': 'queryTimeout', 'type': 'number',
+            'description': 'The maximum duration (In Milliseconds) that an SQL query can execute for.'
+        }
     ]
```

Comment:
Add:
See `sslEnabled` in https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Athena.html

```
    {
        'label': 'SSL Enabled', 'value': 'sslEnabled', 'type': 'checkbox',
        'description': 'Does your database require that you connect to it via SSL? \
                        Note that this is just the connection between this app and your database; \
                        connections to plot.ly or your plotly instance are always encrypted.'
}
```